component/vdr/sidetree/option/recovery/option.go

```diff
@@ -80,16 +80,16 @@ func WithSigningKeyID(id string) Option {
 	}
 }
 
-// WithRevealValue sets reveal value.
-func WithRevealValue(rv string) Option {
+// WithMultiHashAlgorithm set multi hash algorithm for sidetree request.
+func WithMultiHashAlgorithm(multiHashAlgorithm uint) Option {
 	return func(opts *Opts) {
-		opts.RevealValue = rv
+		opts.MultiHashAlgorithm = multiHashAlgorithm
 	}
 }
 
-// WithMultiHashAlgorithm set multi hash algorithm for sidetree request.
-func WithMultiHashAlgorithm(multiHashAlgorithm uint) Option {
+// WithRecoveryCommitment sets recovery commitment.
+func WithRecoveryCommitment(recoveryCommitment string) Option {
```

Comment:
We should probably call this last operation commitment or similar

Reply:
done